external-crates/move/crates/move-analyzer/trace-adapter/src/runtime.ts

```diff
@@ -111,10 +113,10 @@ interface IRuntimeVariable {
 }
 
 /**
- * Describes a stack frame in the runtime and its current state
- * during trace viewing session.
+ * Describes a stack frame in the runtime representing a Move function
+ * call and its current state during trace viewing session.
  */
-interface IRuntimeStackFrame {
+interface IMoveCallStackFrame {
```

Comment:
Runtime now holds a generalization of Move call stack including (optionally) a PTB summary frame and a native command frame, hence this and other renamings